test-utils/src/marginfi_account.rs

```diff
@@ -625,4 +625,39 @@ impl MarginfiAccountFixture {
     pub fn get_size() -> usize {
         mem::size_of::<MarginfiAccount>() + 8
     }
+
+    /// Use the client to send the transfer ix authority transaction
+    pub async fn try_transfer_account_authority(
+        &self,
```

Comment:
`new_authority` should probably be a fn argument here

Reply:
I agree. I seem to break the test every time I add this though lol. 😅